src/templates/layout/default.hbs

```diff
@@ -19,6 +18,9 @@
   <script src="/public/materialize/js/materialize.min.js"></script>
   <script src="/public/anime.min.js"></script>
   <script src="https://use.fontawesome.com/b47cf7f5d9.js"></script>
-  <script src="index.js"></script>
+
+  <script src="scripts/index.js"></script>
+  {{> scripts}}
```

Comment:
Do not include all the scripts on all the pages. This will slow you down significantly on a slow connection